### ISSUE TITLE
AttributeError: module 'execute' has no attribute 'init_session'

### ISSUE THREAD

**Issue by uccmen** [llSourcell/tensorflow_chatbot#10]
Python newbie here and I tried running `app.py` but failed with the error below.
Anyone got the following? How did you fix it?
```
$ python ui/app.py 
Traceback (most recent call last):
  File "ui/app.py", line 31, in <module>
    sess, model, enc_vocab, rev_dec_vocab = execute.init_session(sess, conf='seq2seq_serve.ini')
AttributeError: module 'execute' has no attribute 'init_session'
```

**Comment by HazeHub** [comment]
yup wasn't able to launch app.py but I have noticed there are some interesting lines about it at the end of execute.py that are commented. You should take a look

**Comment by Seiwert** [comment]
I had this error as well. This error means that python cannot find execute.py. The reason for this is because execute.py is up a directory. I never did figure out how to correctly call the import command to where it can see it but if you move all those files out of the ui folder and into the main folder this error will go away. However, if I were you, I would do a google on how to have the import command look at the parent folder.

**Comment by RajathRD** [comment]
https://github.com/suriyadeepan/easy_seq2seq
Follow this repo

**Comment by AJ-sherlock** [comment]
This Error is coz app.py is not able to import execute.py
copy app.py to the directory where execute.py file is.
